docs/extensions-plugins/jupyterlab/quickstart.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Quickstart
+description: Learn how to set up and start using the Pieces CLI.
+---
+
+import CTAButton from "/src/components/CTAButton";
+import { pieces_app } from "/src/lib/const";
+
+# Quickstart
+
+In this tutorial, you'll learn how to set up and start using the Pieces for JupyterLab Extension. We'll cover the installation process and basic usage to help you get up and running quickly.
+
+<CTAButton
+  href={pieces_app.links.extensions.jupyter_lab}
+  label={'Install from PyPI'}
+  icon={'/assets/pieces_logos/white_circle.png'} 
+  iconDark={'/assets/pieces_logos/black_circle.png'} 
+  type={'secondary'}
+/>
+
+## Prerequisites
+
+:::info
+
+In order to use the Pieces JupyterLab Extension, you must have <a target="blank" href="/installation-getting-started/what-am-i-installing">Pieces OS.</a> We also recommend downloading the Pieces for Developers Desktop App.
+
+:::
+
+- [Pieces OS](/installation-getting-started/what-am-i-installing) installed on your machine.
+- [Python](https://www.python.org/downloads/) installed on your machine.
+- [JupyterLab](https://jupyterlab.readthedocs.io/en/latest/getting_started/installation.html) installed on your machine.
+
+## Steps
+
+### Step 1: Install the Pieces JupyterLab Extension in your current environment
+
+1. Make sure JupyterLab is running:
+
+    ```bash
+    jupyter lab
+    ```
+
+1. Install the extension:
+
+    1. If you want to use the JupyterLab UI to install, select the **Extension Manager**, search for `Pieces`, then select **Install** for the `jupyter-pieces` extension.
+
+    1. You can also run the following command from a console or a notebook:
+
+        ```bash
+        pip install jupyter-pieces
+        ```
+
+1. Once installed, refresh the page in your browser.
+
+### Step 2: Use the Pieces JupyterLab Extension
```

Comment:
So on the other guides we have a very simple step here that shows them how to do one thing. I think it's probably better to keep this as a simple last step of the quickstart and then they can dive into the features. What do you think?

Reply:
If the goal is for the reader to ready to use the product, I personally think it is better to show a real world use case. 
The 2 things we measure for usage are saving snippets, and copilot chats - so this aligns with driving those 2 behaviors from our users.

These are still simple steps, but quickly shows the 2 main use cases. They can then dive in further for more details.